client/src/components/LoginInput.js

```diff
@@ -0,0 +1,24 @@
+import styled from '@emotion/styled';
+import Input from '../components/Input';
+
+const LoginInput = styled(Input)`
+  text-align: center;
+  background: white;
+  color: white;
+  font-size: 22px;
+  padding: 17px 0px;
+  margin: 20px 0px;
+  border: 3px #5ce0b9 solid;
+  border-radius: 30px;
+  box-shadow: 0px 15px 30px grey;
+  ::-webkit-input-placeholder {
```

Comment:
just `::placeholder` should be fine here, especially since emotion handles compatibility

Reply:
Done, thanks :)